usr/share/rear/conf/default.conf

```diff
@@ -2411,15 +2411,21 @@ COPY_AS_IS_TSM=( /etc/adsm
 #                  /usr/local/ibm/gsk8* )
 # This example is dated 2017 so it needs to be adapted to newer TSM versions,
 # cf. https://github.com/rear/rear/pull/1566
+# It results a "stripped down" TSM client which could have some functionality missing.
 # Additionally you need to add your custom configuration files like Includes and Excludes files.
-# An alternative method to reduce the size of the TSM client in ReaR recovery system initrd
-# is to exclude what is not needed from the above specified COPY_AS_IS_TSM via COPY_AS_IS_EXCLUDE_TSM.
-# Usually language files (except for the EN_US language) are not needed so for example
+# An alternative method to reduce the size of the TSM client in ReaR recovery system initrd is
+# to exclude what is not needed from the above specified COPY_AS_IS_TSM via COPY_AS_IS_EXCLUDE_TSM
+# to get a slim TSM client without reduced functionality of what can be used in the recovery system.
+# So for example
 # COPY_AS_IS_EXCLUDE_TSM=( /opt/tivoli/tsm/client/ba/bin/*.jar
 #                          /opt/tivoli/tsm/client/ba/bin/*.log*
 #                          /opt/tivoli/tsm/client/*/*/[A-Z][!N]_?? )
-# excludes in particular language file subdirectories except the EN_US language subdirectory,
-# cf. https://github.com/rear/rear/issues/3189#issuecomment-2079811746
+# excludes *.jar files which are only needed for the Java based GUI X11 client
```

Comment:
Cool, maybe we should make this default?

Reply:
If I knew that those COPY_AS_IS_EXCLUDE_TSM
work sufficiently stable for varios TSM versions
I could make this the default.
For now I prefer to be safe and do not exclude
things from TSM.
Perhaps - as a first step - this COPY_AS_IS_EXCLUDE_TSM
coukld be set as default only on POER architecture
because only there it really matters to have a
small TSM client.
But this is not something I will do within this pull request.